api/controllers/ChangelistsController.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+const Changelists = require('../services/Changelists');
+
+module.exports = {
+    async create(req, res) {
+        try {
+            const changelist = await Changelists.create(req.body);
+            return res.json(changelist);
+        } catch (error) {
+            logError(error);
+            return res.serverError(error);
+        }
+    }
+};
```

Comment:
are you using this controller? how?

Reply:
Ah! Turns out I'm not using this controller. I initially added this looking at the other controllers, but there is no need I believe.